packages/apps/hr/src/components/templates/ExpenseHeader/ExpenseHeader.tsx

```diff
@@ -47,6 +53,9 @@ const ExpenseHeader = ({}) => {
           ? user?.activeCompany?.currency?.symbol
           : user?.activeCompany?.currency?.code
       }`}</Text>
+      {!checkNullString(expense?.groundForRefusal) && (
+        <Label message={expense?.groundForRefusal} type="error" />
```

Comment:
Maybe we can add a title before the refusal message "Ground for refusal: " / "Motif de refus: " maybe 🤷🏼